### ISSUE TITLE
When giving samples please keep them consistent

### ISSUE THREAD

**Issue by ngreyes** [MicrosoftDocs/feedback#2584]
I 'm trying to understand SignalR client side and it's very hard to follow examples when the client refers to a method on the server that is no where to be found on the server examples.  Its hard enough that the samples provided are for easy applications when mine is a much more complex one I need to understand every piece of information provided.  Just do diligence on the examples please....

**Comment by rootsmusic** [comment]
@ngreyes Please link to an example.
